app/controllers/books_controller.rb

```diff
@@ -0,0 +1,11 @@
+# Controller to handle book management
+class BooksController < ApplicationController
+  respond_to :json
+
+  def index
+    @page = (params[:page] || 1).to_i
```

Comment:
there's no need to do this, just use params[:page] directly, if it's nil it will query the first page. Also there's no need to cast it to an integer